client/src/main/java/com/vesoft/nebula/client/graph/net/SyncConnection.java

```diff
@@ -192,7 +188,6 @@ public boolean ping() {
             execute(0, "YIELD 1;");
             return true;
         } catch (IOErrorException e) {
-            e.printStackTrace();
             return false;
```

Comment:
please add some log print When ping() returns false， log will help to locate the problem.